src/components/molecules/StartTable/StartTable.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import { useAsyncFn } from "react-use";
+
+import { updateVenueTable } from "api/table";
+
+import { Table } from "types/Table";
+import { VenueTemplate } from "types/venues";
+
+import { currentVenueSelector } from "utils/selectors";
+
+import { useSelector } from "hooks/useSelector";
+
+import { DEFAULT_TABLE_CAPACITY } from "../TablesUserList/TablesUserList";
+
+import "./StartTable.scss";
+
+export interface StartTablePropsType {
+  tables: Table[];
+  newTable: Table;
+}
+
+export const StartTable: React.FC<StartTablePropsType> = ({
+  tables,
+  newTable,
+}) => {
+  const venue = useSelector(currentVenueSelector);
+
+  const [table] = tables;
+  const startTableWidth =
+    table.columns &&
+    venue?.template !== VenueTemplate.jazzbar &&
+    (table.columns + 1) * 55;
+
+  const [, updateTables] = useAsyncFn(async () => {
+    if (!venue?.id) return;
+
+    await updateVenueTable({
+      venueId: venue.id,
+      tableOfUser: newTable,
+      tables: [...tables, newTable],
+      title: newTable.title,
+      capacity: newTable.capacity ?? DEFAULT_TABLE_CAPACITY,
```

Comment:
`capacity` is an optional field for the `Table` type. I can't find the right way to set the default capacity as fallback here

Reply:
The default within the original helper itself? If you pass undefined to an prop, even if that’s via another variable, and it’s defaulted within the function, the default will be used.

Does that solve your need here?

Reply:
looks like I fixed it here 0a1d6fdc0bdb881d1069f3e01f10a99766832366